pom.xml

```diff
@@ -251,15 +251,15 @@
         <version.plugin.maven.dependency>2.3</version.plugin.maven.dependency>
         <version.postgresql>9.1-901.jdbc4</version.postgresql>
         <version.resteasy.api>3.0.12.Final</version.resteasy.api>
-        <version.resteasy>6.1.0.Final</version.resteasy>
+        <version.resteasy>6.2.1.Final</version.resteasy>
```

Comment:
Thanks @mmusgrov for your PR!! That is much appreciated! Please notice the resteasy update is great but it might be omitted for conflict with other narayana dependencies. That is why I opened a PR on narayana repo https://github.com/jbosstm/narayana/pull/2074

Reply:
I thought you said this was one of the options?

Reply:
We need a separate project for the boms (ala WildFly) to avoid this kind of brittle dependency.